apps/result/models.py

```diff
@@ -173,14 +173,14 @@ def language(self):
 
     @property
     def result_short_title_by_language(self):
-        if self.language == "cy" and self.result_short_title_welsh:
+        if self.language == "cy" and self.result_short_title_welsh and self.result_short_title_welsh.strip():
```

Comment:
`self.result_short_title_welsh` can be `None` and `None` doesn't have the `strip()` method on it.